tt_metal/impl/dispatch/command_queue.cpp

```diff
@@ -1470,34 +1479,55 @@ void EnqueueProgramCommand::write_program_command_sequence(const ProgramCommandS
 }
 
 void EnqueueProgramCommand::process() {
+    auto config_sizes = program.get_program_config_sizes();
+    // Require one launch config buffer entry.
+    config_sizes.push_back(1);
```

Comment:
Is there a better way to handle adding the extra entry? This means we copy and append to a vector each time we enqueue a program (though the vector is small so may be minimal impact)